backend/curriculum_tracking/tests/test_AgileCard.py

```diff
@@ -688,3 +687,128 @@ def test_cancel_workshop_attendance(self):
+        self.time_three = self.project_1.start_time + timedelta(days=3)
+        self.time_four = self.project_1.start_time + timedelta(days=2)
+
+    def test_correct_ids_returned_before_or_after_review_request(self):
+
+        # Four reviews are made with the four review times above (No reviews done on project_2)
+        review_1 = factories.RecruitProjectReviewFactory(
+            status=NOT_YET_COMPETENT,
+            recruit_project=self.project_1,
+            timestamp=self.time_one
+        )
+        review_1.timestamp = self.time_one
+        review_1.save()
+
+        review_2 = factories.RecruitProjectReviewFactory(
+            status=COMPETENT,
+            recruit_project=self.project_1,
+        )
+        review_2.timestamp = self.time_two
+        review_2.save()
+
+        review_3 = factories.RecruitProjectReviewFactory(
+            status=EXCELLENT,
+            recruit_project=self.project_1,
+        )
+        review_3.timestamp = self.time_three
+        review_3.save()
+
+        review_4 = factories.RecruitProjectReviewFactory(
+            status=NOT_YET_COMPETENT,
+            recruit_project=self.project_1,
+        )
+        review_4.timestamp = self.time_four
+        review_4.save()
+
+        ids_which_should_be_returned = [review_2.reviewer_user.id, review_3.reviewer_user.id, review_4.reviewer_user.id]
+
+        # Making sure that reviews were done on card and not on card_2, if card_2 returns reviews then our function is
+        # returning the wrong stuff and therefore it is not working as it should.
+        self.assertEqual(
+            sorted(self.card_1.get_users_that_reviewed_since_last_review_request()),
+            sorted(ids_which_should_be_returned)
+        )
+
+    def test_no_user_ids_should_return_since_review_done_before_latest_review_request(self):
+
+        # Create a review request on card_2 of project_2
+        request_review_time_2 = self.project_2.start_time + timedelta(1)
+        self.project_2.request_review(force_timestamp=request_review_time_2)
+        time_of_review = self.project_2.start_time - timedelta(days=10)
+
+        # Create a review on card_2, the review will be before the latest request for a review
+        review_1_card_2 = factories.RecruitProjectReviewFactory(
+            status=NOT_YET_COMPETENT,
+            recruit_project=self.project_2,
+            timestamp=time_of_review
+        )
+        review_1_card_2.timestamp = time_of_review
+        review_1_card_2.save()
+
+        # Review done on project_2, card_2 were done before the latest request for a review, therefore,
+        # get_users_that_reviewed_since_last_review_request() should return no user id's
+        assert len(self.card_2.get_users_that_reviewed_since_last_review_request()) == 0
+
+    def test_two_different_projects_return_different_results_for_users_that_reviewed_since_last_review_request(self):
+
+        # Creating two new projects
```

Comment:
Since this is a method of `ReviewerIdsSinceLatestReviewRequest` we know from the context thats what it's doing. I think `test_two_different_projects_return_different_results` is clear enough.

Reply:
That would be clear enough.